docs/en/changelog.md

```diff
@@ -2,6 +2,27 @@
 
 ## MMSelfSup
 
+### v0.9.2 (28/07/2022)
+
+#### New Features
+
+- Support MAE Reconstructed Image Visualization ([#376](https://github.com/open-mmlab/mmselfsup/pull/376))
+
+#### Bug Fixes
+
+- Fix args/cfg bug in extract.py ([#357](https://github.com/open-mmlab/mmselfsup/pull/357))
+- Fix bugs of SimMIM config ([#360](https://github.com/open-mmlab/mmselfsup/pull/360))
```

Comment:
Better to be more concrete, this modification should be ``fix SimMIM mask generator config bug`` and change the content in Chinese version accordingly.